client/my-sites/plans/current-plan/jetpack-checklist/constants.js

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
This file may have been unintentionally added. Did you mean to edit `client/my-sites/plans/current-plan/jetpack-checklist/constants.ts`?

Reply:
oh yeah, that's weird. I don't know how that even happened. Thanks for catching that.